nextflow_schema.json

```
@@ -5,6 +5,23 @@
     "description": "A pipeline for de novo transcriptome assembly of short reads from bulk RNA-seq",
     "type": "object",
     "definitions": {
+        "workflow_options": {
+            "title": "Workflow options",
```

```suggestion
            "title": "Partial run options",
```
or something similar more descriptive